resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -245,4 +339,57 @@ function DoesRegistryKeyPropertyExist {
     $itemProperty = Get-ItemProperty -Path $Path  -Name $Name -ErrorAction SilentlyContinue
     return $null -ne $itemProperty
 }
+
+function Get-CombinedState {

```

Comment:
Lets rename this function so that it is clear what it is doing. Something like GetAnimationEffectsStateHexValue